src/parser.js

```diff
@@ -0,0 +1,143 @@
+const yargs = require('yargs/yargs');
+const {hideBin} = require('yargs/helpers');
```

Comment:
This dependency is not listed in package.json. You should include package.json and package-lock.json files in your next commit for me to proceed with a review.

Reply:
It seems like I installed it globally. Now I did it in appropriate way.

Reply:
Thanks!